mender-update/daemon/state_test.cpp

```diff
@@ -87,6 +87,132 @@ struct StateTransitionsTestCase {
 	bool device_type_mismatch {false};
 };
 
+
+vector<StateTransitionsTestCase> idle_and_sync_test_cases {
+	StateTransitionsTestCase {
+		.case_name = "Normal_flow_Idle_Sync_Idle",
+		.state_chain =
+			{
+				"Idle_Enter_00",           "Idle_Leave_00",
```

Comment:
Yeh, I agree, it's retarded.

Reply:
I think this should be fixed before submission. In addition to reading, it makes copy-pasting/rearranging very tricky. If you can find an option for it in the clang-format it would be great. But feel free to slap a `clang-format off` on it if you don't feel like dealing with it now.